pages/core-api/safe-installation-guide.mdx

```diff
@@ -0,0 +1,250 @@
+
+#### Hard requirements
+
+- EVM-compatible chain.
+- At least two dedicated RPC node providers, with preference for at least one Tier 1 provider of Infura, Alchemy, Quicknode. You can find more on RPC nodes [here](https://docs.safe.global/core-api/api-safe-transaction-service/rpc-requirements).
+
+### Option 2: Third-party integrators
+
+We are happy to introduce you to our official integrators.
+
+List of official integrators:
+
+    - integrator A | website | telegram | twitter
+    - integrator B | website | telegram | twitter
+    - integrator C | website | telegram | twitter
+
+### Option 3: Self-hosting
+
+If you have engineering and Cloud resources, you can also deploy the different components of the Stack by following one of the procedures below. 
+
+You can find docker images for a majority of platforms. 
+
+#### Requirements
+
+**Hardware**
+
+These are the hardware requirements to run the different services required.
+
+
+| Network | Transaction Service | Client Gateway | Config Service | Database |
+| --- | --- | --- | --- | --- |
+| Small (new chain with low traffic) | CPU: 2 vCPU RAM: 8GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 1 cores/ 2 vCPU RAM: 8GiB |
+| Standard | CPU: 4 vCPU RAM: 16GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 2 cores/ 4 vCPU RAM: 16GiB |
+| Large (large chain with high-traffic) | CPU: 8 vCPU RAM: 32GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 2 vCPU RAM: 8 GiB | CPU: 8 cores/ 16 vCPU RAM: 64GiB |
+
+
+**RPC**
+
+For support, please refer to the [RPC requirements](https://docs.safe.global/core-api/api-safe-transaction-service/rpc-requirements#what-are-the-rpc-requirements). Safe (L1) requires tracing for indexing, while Safe (L2) supports events/logs.
+
+#### Docker-compose deployment
+
+Please see the [safe-infrastructure](https://github.com/safe-global/safe-infrastructure) repository for all required information.
+
+```bash
+git clone git@github.com:safe-global/safe-infrastructure.git
+cd safe-infrastructure
+```
+
+**Prerequisites**
+
+- Docker v20.10+
+- Docker-compose 2.x.x+
+- Running Ethereum JSON RPC client
+
+**Configuration**
+
+```bash
+cp .env.sample .env
+vi .env
+REVERSE_PROXY_PORT=8000
+CFG_VERSION=latest
+CGW_VERSION=latest
+TXS_VERSION=latest
+UI_VERSION=latest
+EVENTS_VERSION=latest
+RPC_NODE_URL=<REPLACE BY YOUR RPC ENDPOINT>
+```
+
+**Run**
+
+This command runs seventeen Docker containers from the [docker-compose.yml](https://github.com/gjeanmart/safe-infrastructure/blob/main/docker-compose.yml):
+- Nginx reverse proxy
+- Postgres 14.x database (x3 for Transaction Service, Config Service, and Events Service)
+- Redis database (x2 for Transaction Service and Client Gateway)
+- RabbitMQ message queue (x2 for Transaction Service and General)
+- Transaction Service (txs) workers & scheduler (x4)
+- Transaction Service (txs) web
+- Config Service (cfg) web
```

Comment:
Can the items in the parathesis cfg, ui, txs, etc can be removed?

Reply:
I guess so. Will remove and ask Greg for feedback.